server.go

```diff
@@ -120,6 +123,8 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 	return
 }
 
+// HandlePacket processes each packet that is sent to the server, and sends to an
+// appropriate worker (EventWorker or Worker).
```

Comment:
I'm adding that as a comment inside the function, rather than in the function description, because I think it's only relevant to people who are modifying `HandlePacket`, correct? (If not, and it's relevant to people who are using veneur, I can add it to the docstring!